doc/develop/test/twister.rst

```diff
@@ -362,6 +362,12 @@ harness_config: <harness configuration options>
     ordered: <True|False> (default False)
         Check the regular expression strings in orderly or randomly fashion
 
+    sysbuild: <True|False> (default False)
+        Build the project using sysbuild infrastructure. Only the main project's
+        generated devicetree and Kconfig will be used for filtering tests.
+        on device testing must use the hardware map, or west flash to load
+        the images onto the target.
+
```

Comment:
This should not be under `harness_config`.

Reply:
Fixed, thanks for catching this